constraints/miden-vm/hash.air

```diff
@@ -181,6 +225,33 @@ ev hasher_state(main: [s[3], h[12], i]):
         is_unchanged(h[j + 4]) for j in 0..4 when !b & f_absorb_node
         h[j + 8]' = h[j + 4] for j in 0..4 when b & f_absorb_node
 
+ev enforce_rpo_round(main: [h[12]]):
```

Comment:
We don't need segment names any more, so here and in the other evaluators we should get rid of them. 

But maybe it will be better to remove them in a future PR, I'm not sure.

Reply:
Done in this PR.